src/cli-sync-ot.ts

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env node
+import { logger } from './logger';
+
+import colors from 'colors';
+import { parseCliSyncOtArguments, createOneTrustGotInstance } from './oneTrust';
+import { OneTrustPullResource } from './enums';
+import { syncOneTrustAssessments } from './oneTrust/helpers/syncOneTrustAssessments';
+import { buildTranscendGraphQLClient } from './graphql';
+
+/**
+ * Pull configuration from OneTrust down locally to disk
+ *
+ * Dev Usage:
+ * yarn ts-node ./src/cli-sync-ot.ts --hostname=customer.my.onetrust.com --oneTrustAuth=$ONE_TRUST_OAUTH_TOKEN --transcendAuth=$TRANSCEND_API_KEY
+ *
+ * Standard usage
+ * yarn cli-sync-ot --hostname=customer.my.onetrust.com --oneTrustAuth=$ONE_TRUST_OAUTH_TOKEN --transcendAuth=$TRANSCEND_API_KEY
+ */
+async function main(): Promise<void> {
+  const {
+    file,
+    fileFormat,
+    hostname,
+    oneTrustAuth,
+    transcendAuth,
+    transcendUrl,
```

Comment:
so transcendAuth + transcendUrl are optional? If they don't pass it in we'll just save to disk?

Reply:
Yes, with a caveat: the criteria for saving to disk is that `--dryRun=true`, in which case we the `parseCliSyncOtArguments` helper also ensures that `transcendAuth` + `transcendUrl` were passed in.

Reply:
Wait i thought dryrun=true would mean don't upload to Transcend

Reply:
Sorry, you're right! I meant `dryRun=false` means ensuring that `transcendAuth` and `transcendUrl` were passed in!